app/scripts/views/components/dataTable/tableView.js

```diff
@@ -76,7 +76,8 @@
           group = dimension.group();
           initPieTableData();
         }
-        initReactTable(true);
+        initReactTable(true, undefined, sequencedSampleIds);
```

Comment:
instead of sending `undefined` as an parameter why not `initReactTable(true, sequencedSampleIds)` moving that `undefined` parameter to last in `initReactTable` implementation